map/map-reference-scale/README.md

```diff
@@ -1,35 +1,35 @@
 # Map reference scale
 
-Set the map's reference scale and choose which feature layers should honor the reference scale.
+Set the map's reference scale and which feature layers should honor the reference scale.
 
-![](MapReferenceScale.png)
+![Image of map reference scale](MapReferenceScale.png)
 
 ## Use case
 
 Setting a reference scale on an `ArcGISMap` fixes the size of symbols and text to the desired height and width at that scale. As you zoom in and out, symbols and text will increase or decrease in size accordingly. When no reference scale is set, symbol and text sizes remain the same size relative to the `MapView`.
 
-Map annotations are typically only relevant at certain scales. For instance, annotations to a map showing a construction site are only relevant at that construction site's scale. So, when the map is zoomed out that information shouldn't scale with the `MapView`, but should instead remain scaled with the `ArcGISMap`.
+Map annotations are typically only relevant at certain scales. For instance, annotations to a map showing a construction site are only relevant at that construction site's scale. So, when the map is zoomed out that information shouldn't scale with the `MapView`, but should instead remain scaled with the `ArcGISMap`. 
 
 ## How to use the sample
 
```

Comment:
this section should be written in paragraph format as per readme template (currently it's numbered)